bld/namelist_files/namelist_defaults_ctsm_tools.xml

```diff
@@ -66,8 +66,8 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <scripgriddata_lrgfile_needed hgrid="3x3min" lmask="nomask">64bit_offset</scripgriddata_lrgfile_needed>
 <scripgriddata hgrid="360x720cru"   lmask="nomask" 
    >lnd/clm2/mappingdata/grids/SCRIPgrid_0.5x0.5_nomask_c110308.nc</scripgriddata>
-<scripgriddata hgrid="0.9x1.25"  lmask="nomask"
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.9x1.25_nomask_c191014.nc</scripgriddata>
+<!--scripgriddata hgrid="0.9x1.25"  lmask="nomask"
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_0.9x1.25_nomask_c191014.nc</scripgriddata-->
```

Comment:
I commented this line out because the actual file in use is `0.9x1.25_c110307.nc`